e2e/tests/LoggedIn/publish.e2e.spec.ts

```diff
@@ -837,12 +839,14 @@ const checkPublicationOnAccountPage = async (
             await expect(publicationContainer).toContainText(
                 'Someone else is working on a new draft version, and you do not yet have access to it'
             );
+            await expect(publicationContainer.locator(PageModel.myAccount.requestControlButton)).toBeVisible();
```

Comment:
Same as above comment, could be helpful to have a test to confirm this one doesn't show on other states

Reply:
Agreed, have pushed a commit that specifies the scenarios when viewing as a coauthor in which this button should not be shown.